fmrest/server.py

```diff
@@ -853,6 +853,50 @@ def get_layout_metadata(self, layout: Optional[str] = None) -> Dict:
 
         return response
 
+    def get_layout_valueList(self, name: str, layout: Optional[str] = None):
+        """Fetch layout metadata and return the 'name' valueList
+        suitable to use in a Form Select (tuple)
+        Retrieves layout metadata and returns a list of "name" values
+        for use in a form SELECT (tuple)
+
+        Example:
+            values = fms.get_layout_valueList()
```

Comment:
Missing name parameter in example: `fms.get_layout_valueList('myListValue', 'myOptionalLayoutName')`